### PR TITLE
Bug 1688633-1689757 Samples Operator feedback

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1689757
https://bugzilla.redhat.com/show_bug.cgi?id=1688633

### CODE REVIEW

**Comment by xiuwang** [comment]
The change lgtm, but I have one question, The part has been removed from offical doc https://docs.openshift.com/container-platform/4.1/openshift_images/configuring-samples-operator.html#configuration-resource-overview_configuring-samples-operator ?

**Comment by bmcelvee** [comment]
Thanks, @xiuwang!

@openshift/team-documentation please peer review

File: modules/samples-operator-configuration.adoc [hunk 1]
```diff
@@ -43,13 +49,20 @@ Creation or update of RHEL content is not gated by the existence of the pull
 secret if the `Samples Registry` is overridden to a value other than the empty
 string or registry.redhat.io.
 ====
-+
-* *Architectures:* Place holder to choose an architecture type. Currently only x86
+
+|`Architectures`
+|Place holder to choose an architecture type. Currently only x86
```
Comment (bergerhoffer) [inline modules/samples-operator-configuration.adoc]: placeholder is one word.

File: modules/samples-operator-configuration.adoc [hunk 2]
```diff
@@ -70,33 +83,50 @@ The Samples Operator still processes secrets while in `Removed` state. You can
...
-** `True` when imagestreams are created/updated, but
+[cols="3a,8a",options="header"]
+|===
+|Condtion |Description
+
+|`SamplesExists`
+|Indicates the samples are created in the OpenShift namespace.
+
+|`ImageChangesInProgress`
+|`True` when imagestreams are created/updated, but
```
Comment (bergerhoffer) [inline modules/samples-operator-configuration.adoc]: minor, but suggest to change "created/updated" to "created or updated"

File: modules/samples-operator-configuration.adoc [hunk 3]
```diff
@@ -70,33 +83,50 @@ The Samples Operator still processes secrets while in `Removed` state. You can
...
-    pending, but are waiting for in progress imagestreams to complete.
-* *ImportImageErrorsExist.* Indicator of which imagestreams had errors during
-    the image import phase for one of their tags.
-** `True` when an error has occurred. The list of imagestreams with an error is
+
+|`ImportCredentialsExist`
+|A `samples-registry-credentials` secret is copied into the OpenShift namespace.
+
+|`ConfigurationValid`
+|`True` or `false` based on whether any of the restricted changes noted
```
Comment (bergerhoffer) [inline modules/samples-operator-configuration.adoc]: Capitalization is different between True and false. I assume it's case sensitive and they should be the same?

File: modules/samples-operator-configuration.adoc [hunk 4]
```diff
@@ -70,33 +83,50 @@ The Samples Operator still processes secrets while in `Removed` state. You can
...
+previously are submitted.
+
+|`RemovePending`
+|Indicator that there is a`Management State: Removed` setting pending, but are
```
Comment (bergerhoffer) [inline modules/samples-operator-configuration.adoc]: Need a space after "a", the backticks are rendering.

File: modules/samples-operator-configuration.adoc [hunk 5]
```diff
@@ -70,33 +83,50 @@ The Samples Operator still processes secrets while in `Removed` state. You can
...
 
 The samples resource maintains the following conditions in its status:
 
-* *SamplesExists.* Indicates the samples are created in the OpenShift
-namespace.
-* *ImageChangesInProgress.*
-** `True` when imagestreams are created/updated, but
+[cols="3a,8a",options="header"]
+|===
+|Condtion |Description
```
Comment (bergerhoffer) [inline modules/samples-operator-configuration.adoc]: Condtion -> Condition

**Comment by bmcelvee** [comment]
Thanks so much, @bergerhoffer! :cat:

File: modules/samples-operator-configuration.adoc [hunk 6]
```diff
@@ -70,33 +83,50 @@ The Samples Operator still processes secrets while in `Removed` state. You can
...
-    pending, but are waiting for in progress imagestreams to complete.
-* *ImportImageErrorsExist.* Indicator of which imagestreams had errors during
-    the image import phase for one of their tags.
-** `True` when an error has occurred. The list of imagestreams with an error is
+
+|`ImportCredentialsExist`
+|A `samples-registry-credentials` secret is copied into the OpenShift namespace.
+
+|`ConfigurationValid`
+|`True` or False` based on whether any of the restricted changes noted
```
Comment (xiuwang) [inline modules/samples-operator-configuration.adoc]: Missing a ` before word 'False'

**Pull request merged**